src/components/clusters/PullSecret.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import { Checkbox, FormGroup, Popover } from '@patternfly/react-core';
+import { ExternalLinkAltIcon, OutlinedQuestionCircleIcon } from '@patternfly/react-icons';
+import { ClusterCreateParams, ocmClient } from '../../api';
+import { CLUSTER_MANAGER_SITE_LINK, PULL_SECRET_INFO_LINK } from '../../config';
+import { TextAreaField } from '../ui';
+import { useFormikContext } from 'formik';
+
+export type PullSecretProps = {
+  pullSecret?: string;
+};
+
+const ClusterManagerSiteLink = () => (
+  <a href={CLUSTER_MANAGER_SITE_LINK} target="_blank" rel="noopener noreferrer">
+    OpenShift Cluster Manager <ExternalLinkAltIcon />
+  </a>
+);
+
+const PullSecretInfoLink = () => (
+  <a href={PULL_SECRET_INFO_LINK} target="_blank" rel="noopener noreferrer">
+    Learn more about pull secrets <ExternalLinkAltIcon />.
+  </a>
+);
+
+const pullSecretHelperText = ocmClient ? (
+  <>
+    Your Red Hat account's pull secret is used by default.&nbsp;
+    <PullSecretInfoLink />
+  </>
+) : (
+  <>
+    A Red Hat account's pull secret can be found in &nbsp;
+    <ClusterManagerSiteLink />
+  </>
+);
+
+const PullSecretInfo = () => (
+  <Popover
+    position="top"
+    bodyContent={
+      <>
+        Pull secrets are used to download OpenShift Container Platform components and connect
+        clusters to a Red Hat account.
+        {ocmClient ? (
+          <PullSecretInfoLink />
+        ) : (
+          <>
+            Pull secrets can be found in&nbsp;
+            <ClusterManagerSiteLink />
+          </>
+        )}
+      </>
+    }
+  >
+    <OutlinedQuestionCircleIcon />
```

Comment:
```suggestion
    <button onClick={(e) => e.preventDefault()} className="pf-c-form__group-label-help">
      <OutlinedQuestionCircleIcon noVerticalAlign />
    </button>
```
(according to https://www.patternfly.org/v4/components/form#basic)

We could eventually extract this into a specific `<FormFieldHelpPopover>` component.

Reply:
This change ensures correct icon size and alignment.